packages/manager/apps/hub-next/src/main.ts

```diff
@@ -0,0 +1,37 @@
+import { createApp } from 'vue';
+import { createI18n } from 'vue-i18n';
+
+import { fetchConfiguration, Environment, LANGUAGES } from '@ovh-ux/manager-config';
+import { loadLocaleMessages } from '@/i18n';
+
+// css
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '@ovh-ux/ui-kit/dist/css/oui.css';
+
+import App from './App.vue';
+import router from './router';
+import store from './store';
+
+fetchConfiguration('hub-next').then(async () => {
```

Comment:
Not sure we'll be able to have the correct app returned with this, but not sure about conflicts if we register `hub` twice

Reply:
you could call it with `fetchConfiguration('hub')`. This application stands for the `hub` application.